testing/matrix-sdk-integration-testing/src/tests/redaction.rs

```diff
@@ -34,12 +34,12 @@ async fn test_redacting_name() -> Result<()> {
     let room_id = room.room_id().to_owned();
     for _ in 0..=10 {
         sync_token = Some(sync_once(&tamatoa, sync_token).await?);
-        if tamatoa.get_joined_room(&room_id).is_some() {
+        if tamatoa.get_room(&room_id).is_some() {
             break;
         }
     }
 
-    let room = tamatoa.get_joined_room(&room_id).unwrap();
+    let room = tamatoa.get_room(&room_id).unwrap();
```

Comment:
here too etc.

Reply:
This is a full integration test that talks to a real server, the server will error further down when you're not actually joined.